test/support/helpers/session.helper.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+/**
+ * @module SessionHelper
+ */
+
+const { generateUUID } = require('../../../app/lib/general.lib.js')
+const SessionModel = require('../../../app/models/session.model.js')
+
+/**
+ * Add a new session
+ *
+ * If no `data` is provided, default values will be used. These are
+ *
+ * - `id` - [random UUID]
+ * - `data` - [empty object {}]
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here or in the database
+ *
+ * @returns {module:SessionModel} The instance of the newly created record
+ */
+function add (data = {}) {
+  const insertData = defaults(data)
+
+  return SessionModel.query()
+    .insert({ ...insertData })
+    .returning('*')
+}
+
+/**
+ * Returns the defaults used
+ *
+ * It will override or append to them any data provided. Mainly used by the `add()` method, we make it available
+ * for use in tests to avoid having to duplicate values.
+ *
+ * @param {Object} [data] Any data you want to use instead of the defaults used here in the database
+ */
+function defaults (data = {}) {
+  const defaults = {
+    id: generateUUID(),
+    data: {}
+  }
```

Comment:
Where a table automatically generates an ID we don't tend to set one in the helper. If you've seen us do this it should normally come with a note saying the previous team failed to implement an automatic UUID on insert in the underlying table, hence the need to do it.

So, we just focus on setting meaningful defaults to reflect what an instantiated record would typically look like.

In this particular case `data: {}` could be what we often find. So, I'd suggest one of the following, I'll leave you to decide which one to go with!

```javascript
  const defaults = {
    data: { licenceId: '01/128' }
  }
```

> Or whatever pretend test values you'd like to set in `data:`

or 


```javascript
  const defaults = {}
```